cypress/integration/notyf_spec.js

```diff
@@ -370,6 +370,20 @@ context('Notyf', () => {
   });
 
   describe('Public API', () => {
+
+    it('should dismiss one notification', () => {
```

Comment:
According to the comment about not touching the global directly, this test case would need some adjustment. You can type the index in the text input, and then click on "dismiss" to achieve the same effect.